mongo/options/collectionoptions.go

```diff
@@ -13,45 +13,56 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/writeconcern"
 )
 
-// CollectionOptions represent all possible options to configure a Collection.
+// CollectionOptions represents options that can be used to configure a Collection.
 type CollectionOptions struct {
-	ReadConcern    *readconcern.ReadConcern   // The read concern for operations in the collection.
-	WriteConcern   *writeconcern.WriteConcern // The write concern for operations in the collection.
-	ReadPreference *readpref.ReadPref         // The read preference for operations in the collection.
-	Registry       *bsoncodec.Registry        // The registry to be used to construct BSON encoders and decoders for the collection.
+	// The read concern to use for operations executed on the collection. The default value is nil, which means that
+	// the read concern of the database used to create the collection will be used.
```

Comment:
Maybe it's just me, but I'm reading "create the collection" incorrectly as referring to the act of creating the collection in MongoDB itself.  Maybe because "collection" is lowercase? 

Consider rephrasing.

Reply:
Changed all docs for this type to say `database used to configure the Collection` instead.